src/org/traccar/MainEventHandler.java

```diff
@@ -34,14 +35,17 @@
 public class MainEventHandler extends ChannelInboundHandlerAdapter {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(GeocoderHandler.class);
+    private static final String DEFAULT_LOGGER_EVENTS = "time,position,speed,course,accuracy,result";
 
     private final Set<String> connectionlessProtocols = new HashSet<>();
+    private final Set<String> logEvents;
 
     public MainEventHandler() {
         String connectionlessProtocolList = Context.getConfig().getString("status.ignoreOffline");
         if (connectionlessProtocolList != null) {
             connectionlessProtocols.addAll(Arrays.asList(connectionlessProtocolList.split(",")));
         }
+        logEvents = Sets.newHashSet(Context.getConfig().getString("logger.events", DEFAULT_LOGGER_EVENTS).split(","));
```

Comment:
I would prefer to use standard Java API instead and minimize dependencies usage when possible.